[rocprofiler-compute] Threshold Based Clamping in Analyze Stage (#2565)
ROCm/rocm-systems: projects/rocprofiler-compute/docs/reference/faq.rst

When ROCm Compute Profiler detects values exceeding their theoretical peaks, it displays a warning message indicating this behavior.

What does "Counter variance corrected" mean?
=============================================

> When profiling, you may see a warning like:

Reviewer: ```suggestion
When profiling, you may see the following warning:
```
Minor phrasing adjustment for user clarity- state that the warning will be exactly as is, and not infer it will look 'like' the warning below
